cmake/Modules/CocosBuildHelpers.cmake

```diff
@@ -172,27 +178,19 @@ endfunction()
 function(setup_cocos_app_config app_name)
     # set target PROPERTIES, depend different platforms
     if(APPLE)
-        set(APP_BIN_DIR "${CMAKE_BINARY_DIR}/bin")
         set_target_properties(${app_name} PROPERTIES MACOSX_BUNDLE 1
                               )
     elseif(MSVC)
-        # only Debug and Release mode was supported when using MSVC.
-        set(APP_BIN_DIR "${CMAKE_BINARY_DIR}/bin/${APP_NAME}/$<CONFIG>")
-        set(APP_RES_DIR "${CMAKE_BINARY_DIR}/bin/${APP_NAME}/${CMAKE_BUILD_TYPE}/Resources")
         #Visual Studio Defaults to wrong type
         set_target_properties(${app_name} PROPERTIES LINK_FLAGS "/SUBSYSTEM:WINDOWS")
```

Comment:
I think that we should remove it.
In modern cmake compiler/linker should not be set.
User should be able to setup flags when needed in root CMakeLists.txt.

Reply:
I will check this

Reply:
convert it into
```
# visual studio default is Console app, but we need Windows app
target_link_options(${app_name} PRIVATE "/SUBSYSTEM:WINDOWS")
```
so it didn't disturb user set flags in root CMakeLists.txt